cmd/explaintest/run-tests.sh

```diff
@@ -22,6 +22,7 @@ importer=""
 tidb_server=""
 portgenerator=""
 explain_test_log="./explain-test.out"
+explain_test_disable_new_collation_log="./explain-test-disable-new-collation.out"
```

Comment:
Why not use `>>` redirecting the logs into `./explain-test.out`?

So you don't have to update [the CI script](https://github.com/PingCAP-QE/ci/blob/main/jenkins/pipelines/ci/tidb/tidb_ghpr_check.groovy#L175).